src/bin/www.ts

```diff
@@ -0,0 +1,39 @@
+import Koa = require('koa');
+import {productsRouter} from '../routers/product-router';
+import bodyParser = require('koa-bodyparser');
+import mongoose = require('mongoose');
+import {logger, loggerMiddleware} from '../middlewares/logger';
+import {config} from '../configurations/configuration';
+import {errorHandlerMiddleware} from '../middlewares/error-handler';
+
+const connectToDB = async () => {
+    try {
+        logger.info(config.get('dbURI'));
+        await mongoose.connect(config.get('dbURI'), {
```

Comment:
should not be part of www, create function that connects to db and listen to the events

Reply:
seperated it to db-connector file